src/main.rs

```diff
@@ -1254,7 +1254,9 @@ fn cmd_vet(out: &mut dyn Write, cfg: &Config) -> Result<(), VetError> {
         }
 
         // Check if any of our first-parties are in the crates.io registry
-        check_audit_as_crates_io(out, cfg, &store)?;
+        if check_audit_as_crates_io(out, cfg, &store).is_err() {
+            panic_any(ExitPanic(-1));
```

Comment:
Hmm, I don't love that this means we swallow any errors e.g. from `Cache::acquire` here. Could we make the test catch the panic and handle it that way instead?